skbio/sequence/distance.py

```diff
@@ -113,3 +104,54 @@ def hamming(seq1, seq2):
         distance = scipy.spatial.distance.hamming(seq1.values, seq2.values)
 
     return float(distance)
+
+
+@experimental(as_of='0.4.2')
+def kmer_distance(seq1, seq2, k=3, overlap=True):
```

Comment:
IAB uses a default of `k=3` but I think this should be a required parameter to match `Sequence.iter_kmers` and `Sequence.kmer_frequencies`.